pkg/domain/entities/network.go

```diff
@@ -41,19 +41,26 @@ type NetworkRmReport struct {
 
 // NetworkCreateOptions describes options to create a network
 type NetworkCreateOptions struct {
-	DisableDNS bool
-	Driver     string
-	Gateways   []net.IP
-	Internal   bool
-	Labels     map[string]string
-	MacVLAN    string
-	Ranges     []string
-	Subnets    []string
-	IPv6       bool
+	DisableDNS        bool
+	NetworkDNSServers []string
```

Comment:
Can this be alphabetically ordered? It was already a bit out of order before (IPv6 didn't fit), but fixing that is easy enough